app/models/email_access_token.rb

```diff
@@ -5,6 +5,8 @@
 #  id            :bigint           not null, primary key
 #  email_address :citext           not null
 #  token         :string           not null
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
```

Comment:
! Thanks for adding the timestamps. We should make sure that as a rule all of our models are initialized with t.timestamps. IIRC, old versions of rails used to do this by default. Or maybe the migration gets initialized with t.timestamps already there if you name the migration in the "create_table" syntax? I can't remember.

Reply:
Yes! BG and I realized this was missing and knew I'd follow up with this PR. :)